src/components/notification/inline-notification.hbs

```diff
@@ -1,44 +1,43 @@
 <div data-notification class="bx--inline-notification bx--inline-notification--error" role="alert">
   <div class="bx--inline-notification__details">
-    <svg class="bx--inline-notification__icon" width="16" height="16" viewBox="0 0 16 16">
-      <path d="M8 16A8 8 0 1 1 8 0a8 8 0 0 1 0 16zM4 4.65L11.35 12l.65-.65L4.65 4 4 4.65z" fill-rule="evenodd" />
+    <svg class="bx--inline-notification__icon" idth="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
it doesn't seem to affect the actual svg in the component, but just noticed that this one is missing a `w`

Reply:
will fix